README.md

```diff
@@ -1,33 +1,25 @@
 # Random Spotify Song
 
-This project contains Python files that will play a random song from Spotify by looking up the top thirty songs for a random three-letter string or commonly used word and picking one of them.
+## How to use:
 
-In order to allow the program to authenticate with Spotify, you will need to install the `spotipy` module for Python. You can install the latest version with the command:
+This project allows you to play a random song from Spotify by looking up the top thirty songs for a random three-letter string (char schema) or commonly used word and picking one of them (word schema).
 
-```
-pip install git+https://github.com/plamere/spotipy.git --upgrade
-```
+To run the program, double click on `RandomSpotifySong.bat`. There is also a shortcut file which can be moved from the original folder and runs the same as `RandomSpotifySong.bat`.
```

Comment:
This is great! `.bat` only work in DOS-based OS systems. Could you add instructions for non-DOS systems for executing the Python file directly?